community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -122,8 +122,9 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
             },
             EMPTY );
 
-    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections" )
-    Setting<File> certificates_directory = BoltKernelExtension.Settings.certificates_directory;
+    @Description( "Directory for storing certificates to be used by Neo4j for TLS connections. Certificates files must be named neo4j.cert and neo4j.key" )
```

Comment:
`Certificates files` => `Certificate files`
Files and paths with italics (surrounded by underscore):
`neo4j.cert` => `_neo4j.cert_`
`neo4j.key` => `_neo4j.key_`